EveryonesInvited/website
# Eptc

### What changes have you made?

- EsLint + Prettier + Typescript ready

### Which issue(s) does this PR fix?

Fixes #4

### Screenshots (if there are design changes)

### How to test

- typescript warning unchanged, check this branch commit history for test file. If does not work from pull, try restarting vscode
